ADC_Joystick_RTApp_MT3620_BareMetal/CMakeSettings.json

```diff
@@ -1,4 +1,4 @@
-﻿{
+{
```

Comment:
This file could have similar changes to settings.json, removing all usage of Target API set in build directories and in variables.  Also, fwiw, the way that MS HelloWorld RT sample sets up the CMake Toolchain file is different: using "cmakeToolchain" instead of a variable.